requirements/base.in

```diff
@@ -6,3 +6,7 @@ django
 pymongo>=2.7.2,<4.0.0
 pytz
 six
+celery
+django-model-utils
+edx-django-utils
+json-tricks
```

Comment:
While this library is very clever it reaches into the innards of an object in order to figure out how to serialize it and that seems like a level of complexity that is not necessary to solve the problem it's solving.  I'd rather re-generate any python objects from input params than try to do this.